buildtools/docker_rig/start_tari_app.sh

```
@@ -0,0 +1,39 @@
+WAIT_FOR_TOR=${WAIT_FOR_TOR:-0}
+TARI_BASE=${TARI_BASE:-/var/tari/$APP_NAME}
+TARI_CONFIG=${TARI_CONFIG:-/var/tari/config/config.toml}
+USER_ID=${USER_ID:-1000}
+GROUP_ID=${GROUP_ID:-1000}
+
+echo "Starting ${APP_NAME} with following docker environment:"
+echo "executable: ${APP_EXEC}"
+echo "WAIT_FOR_TOR: $WAIT_FOR_TOR"
+echo "base folder (in container): ${TARI_BASE}"
+echo "config file (in container): ${TARI_CONFIG}"
+echo "arguments (in container): ${@}"
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)